Marlin/Marlin_main.cpp

```diff
@@ -7499,11 +7499,11 @@ inline void gcode_M115() {
       SERIAL_PROTOCOLLNPGM("Cap:SOFTWARE_POWER:0");
     #endif
 
-    // TOGGLE_LIGHTS (M355)
+    // CASE LIGHTS (M355)
     #if HAS_CASE_LIGHT
-      SERIAL_PROTOCOLLNPGM("Cap:TOGGLE_LIGHTS:1");
+      SERIAL_PROTOCOLLNPGM("Cap:CASE_LIGHT_BRIGHTNESS:1");
     #else
-      SERIAL_PROTOCOLLNPGM("Cap:TOGGLE_LIGHTS:0");
+      SERIAL_PROTOCOLLNPGM("Cap:CASE_LIGHT_BRIGHTNESS:0");
```

Comment:
@repetier Have you already been following this? I already documented `Cap:TOGGLE_LIGHTS` on the RepRap wiki so maybe you already incorporated it…? Sorry there's so much mucking with these things immediately after release. I thought these features were more developed.

@Bob-the-Kuhn I'm not in agreement with dropping the separate on/off function. I think it makes good sense to keep them separate, so that the light may be toggled off and on without needing to specify a brightness every time. Imagine a case light interface in which rotating the dial sets the brightness, while clicking toggles the on/off stater. When toggling that on/off state, it should retain the brightness that was last set.